### PR TITLE
Autoload traits and suppress log output in tests

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

Test suite changes only:

- Autoload traits and classes for test suite
- Pipe `error_log` output to `/dev/null` instead of `STDOUT`
- Minor composer script changes

### Testing

No changes to test coverage.

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * @package WP-Auth0
  */
 
+ini_set( 'error_log', '/dev/null' );
```
Comment (damieng) [inline tests/bootstrap.php]: Does this mean users and CI can no longer report the errors?  Or do they still go somewhere useful as well?

File: tests/testConstantSettings.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
  */
 class TestConstantSettings extends WP_Auth0_Test_Case {
 
-	use domDocumentHelpers;
+	use DomDocumentHelpers;
```
Comment (damieng) [inline tests/testConstantSettings.php]: Did an external dependency change the casing on this?

Comment (joshcanhelp) [reply tests/testConstantSettings.php]: No, was having trouble with the auto-loader picking up the file.

File: tests/bootstrap.php (hunk 1 above)
Comment (joshcanhelp) [reply tests/bootstrap.php]: `error_log` is not tested currently ... or really very useful. It's legacy functionality that still remains in a handful of API calls. We have our own error log in the plugin that reports issues. 

In short: it was not used for testing and would just output needlessly to the console.

**Pull request merged**